data/shaders/standard.fs.glsl

```diff
@@ -9,5 +9,8 @@ out vec4 o_FragColor;
 
 void main()
 {
-	o_FragColor = texture(u_Texture, v_TextureCoord) * v_QuadColor;
+	
+	vec4 tex = texture(u_Texture, v_TextureCoord);
+	if(tex.a < 0.99) discard;
```

Comment:
do we need this discard?

Reply:
Sadly, for now. I'm looking into it, and it might be an issue with geometry instantiation (I'd check without it but it's quite a rewrite, so I'm going for speed of delivery here), but it should be noted that it's not here to stay, but also is a fully valid shader language thing you shouldn't really look badly upon :P